public/bz_support.js

```diff
@@ -575,6 +575,15 @@ function BZ_SaveMagicField(field_name, field_value, optional, type, answer) {
         var offlineWarning = document.getElementById("bz-offline-warning");
         if(offlineWarning)
           offlineWarning.style.display = "none";
+
+        var responseJson = http.responseText;
+        console.log(responseJson);
+        var res = JSON.parse(responseJson);
+        if(res["points_reason"] == "past_due") {
+          var lateWarning = document.getElementById("bz-late-warning");
+          if(lateWarning)
+            lateWarning.style.display = "none";
```

Comment:
I'm confused about this. This code seems to hide the late warning on the next successful save of a magic field if the module is past due. I don't see any code to show the late warning though. What am I missing?